airflow/serialization/serialized_objects.py

```diff
@@ -252,6 +252,14 @@ def _serialize(cls, var: Any) -> Any:  # Unfortunately there is no support for r
 
     # pylint: enable=too-many-return-statements
 
+    @classmethod
+    def _serialize_and_sort(cls, var):
+        """Serialized and Sort the values in the iterator"""
```

Comment:
This is passed much more than just an iterator, so this comment needs updating/expanding please.

Reply:
hmm 🤔  -- we only pass an iterator no ? 

Exmaple:

```python
_serialize_and_sort([2, "x"])
```

or

```python
_serialize_and_sort((2, "x"))
```

Reply:
Ah, iterable then.

Reply:
oh yeah

Reply:
https://docs.python.org/3/library/stdtypes.html#iterator-types <-- you didn't mean one of these.

Reply:
Updated in https://github.com/apache/airflow/pull/15395/commits/237af383107ac492a486770eb269f27c3769bc5d